### PR TITLE
feat: allow disabling cors

### PR DESCRIPTION
**Pull request by thisisnithin**
## TODO

- [x] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

File: router/pkg/config/config.schema.json [hunk 1]
```diff
@@ -591,6 +591,10 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
+        "enabled": {
```
Comment (StarpTech) [inline router/pkg/config/config.schema.json:594]: Please add a description.

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: router/pkg/config/config.schema.json (hunk 1 above)
Comment (thisisnithin) [reply router/pkg/config/config.schema.json:594]: Done

**Review by StarpTech — APPROVED** [review]
LGTM

**Comment by StarpTech** [comment]
Please don't forget to update the docs.

**Pull request merged**